### DISCUSSION TITLE
Lesson 6 : Problem with MockV3Aggregator.sol

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#1226] [Q&A]
Hello, 
I import the [MockV3Aggregator.sol](https://github.com/smartcontractkit/chainlink-mix/blob/main/contracts/test/MockV3Aggregator.sol )

And I got an error on VSC ( a red underline ) at the line 4.

```
import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV2V3Interface.sol";
```

Is it important ?

**Comment by cromewar** [comment]
Please share the error message here
